Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -1178,16 +1176,6 @@ internal static void SendSpawnMessage(NetworkIdentity identity, NetworkConnectio
         /// <param name="conn">The connections object to clean up for.</param>
         public static void DestroyPlayerForConnection(NetworkConnection conn)
         {
-            // => destroy what we can destroy.
```

Comment:
do we still destroy those objects somewhere now that this code is gone?
if yes, where?
if no, why not?

Reply:
This code was moved to NetworkConnectionToClient in this same PR.

Reply:
but there is no direct call to DestroyOwnedObjects here to call it.
looks like Paul calls DestroyOwnedObjects in conn.Disconnect. 
so I am wondering what could break if this is called at another time, instead of right here

Reply:
Destroy Player for connection is _only_ called OnServerDisconnect(),   which is called when you disconnect the connection.

If we already do this when we disconnect the connection,  then this does nothing at all.